### PR TITLE
Forum post text overflow fix (#4048)

### PR DESCRIPTION
**Pull request by chrlschwb**
#4048

### CODE REVIEW

**Review by traumschule — APPROVED** [review]
LGTM

links

File: packages/ui/src/common/components/MarkdownPreview/MarkdownPreview.tsx [hunk 1]
```diff
@@ -43,7 +43,7 @@ export const MarkdownPreview = ({ markdown, append, ...styleProps }: MarkdownPre
     const p: Components['p'] = ({ children, sourcePosition }) => (
       <p>
         {children}
-        {shouldAppend(sourcePosition) && <> {append}</>}
+        {shouldAppend(sourcePosition) && <p> {append}</p>}
       </p>
```
Comment (thesan) [inline packages/ui/src/common/components/MarkdownPreview/MarkdownPreview.tsx:47]: I completely forgot what this `append` is about but having a `p` element inside another `p` element is considered incorrect html. As a result react will show a warning in development mode. Is this change really necessary ?

File: packages/ui/src/common/components/MarkdownPreview/MarkdownPreview.tsx [hunk 2]
```diff
@@ -59,7 +59,7 @@ export const MarkdownPreview = ({ markdown, append, ...styleProps }: MarkdownPre
             {children}
           </Mention>
         ) : (
-          <a style={{ overflowWrap: 'break-word' }} href={href}>
+          <a style={{ overflowWrap: 'anywhere' }} href={href}>
```
Comment (thesan) [inline packages/ui/src/common/components/MarkdownPreview/MarkdownPreview.tsx:62]: Good solution but please add it in the `MarkdownPreviewStyles` instead of inline.

Also could you try to add it at the `.markdown-preview *` level ? Because even though it's unlikely this issue would occur on anything else than a hyperlink it still might. Eg on a long inline code like this: `firstValueFrom(api.query.referendum.accountsOptedOut.keys()).then((result)`

**Review by thesan — CHANGES_REQUESTED** [review]
